WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentService.swift

```diff
@@ -104,7 +104,11 @@ final class CardPresentPaymentService: CardPresentPaymentFacade {
     func disconnectReader() async {
         readerConnectionStatusSubject.send(.disconnecting)
 
-        cancelPayment()
+        do {
+            try await cancelPayment()
+        } catch {
+            DDLogError("Attempting to cancel the payment has failed \(error)")
```

Comment:
We can log the .couldNotCancelPayment case here, but I don't think we want to throw, right? Both success or failure when cancelling the payment should still permit us to attempt reader disconnection.